sdk/cognitivelanguage/Azure.AI.Language.Conversations/api/Azure.AI.Language.Conversations.netstandard2.0.cs

```diff
@@ -1,258 +1,19 @@
-        public override string ToString() { throw null; }
-    }
-    public partial class AudioTiming
-    {
-        public AudioTiming() { }
-        public long? Duration { get { throw null; } set { } }
-        public long? Offset { get { throw null; } set { } }
-    }
-    public partial class BaseExtraInformation
-    {
-        internal BaseExtraInformation() { }
-    }
-    public partial class BasePrediction
-    {
-        internal BasePrediction() { }
-        public Azure.AI.Language.Conversations.ProjectKind ProjectKind { get { throw null; } set { } }
-        public string TopIntent { get { throw null; } }
-    }
-    public partial class BaseResolution
-    {
-        internal BaseResolution() { }
-    }
-    public partial class BooleanResolution : Azure.AI.Language.Conversations.BaseResolution
-    {
-        internal BooleanResolution() { }
-        public bool Value { get { throw null; } }
-    }
     public partial class ConversationAnalysisClient
     {
         protected ConversationAnalysisClient() { }
         public ConversationAnalysisClient(System.Uri endpoint, Azure.AzureKeyCredential credential) { }
         public ConversationAnalysisClient(System.Uri endpoint, Azure.AzureKeyCredential credential, Azure.AI.Language.Conversations.ConversationAnalysisClientOptions options) { }
+        public virtual System.Uri Endpoint { get { throw null; } }
         public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
         public virtual Azure.Response AnalyzeConversation(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
-        public virtual Azure.Response<Azure.AI.Language.Conversations.AnalyzeConversationTaskResult> AnalyzeConversation(string utterance, Azure.AI.Language.Conversations.ConversationsProject project, Azure.AI.Language.Conversations.AnalyzeConversationOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> AnalyzeConversationAsync(Azure.Core.RequestContent content, Azure.RequestContext context = null) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.AI.Language.Conversations.AnalyzeConversationTaskResult>> AnalyzeConversationAsync(string utterance, Azure.AI.Language.Conversations.ConversationsProject project, Azure.AI.Language.Conversations.AnalyzeConversationOptions options = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public static string GetAuthorizationHeader() { throw null; }
+        public virtual Azure.Operation<System.BinaryData> CancelJob(Azure.WaitUntil waitUntil, System.Guid jobId, Azure.RequestContext context = null) { throw null; }
```

Comment:
Curious, does this LRO return a value once complete?

Reply:
Yes, it uses the same jobs status check.